src/Protocol/Protocol.h

```diff
@@ -86,6 +86,7 @@ class cProtocol
 	virtual void SendExplosion                  (double a_BlockX, double a_BlockY, double a_BlockZ, float a_Radius, const cVector3iArray & a_BlocksAffected, const Vector3d & a_PlayerMotion) = 0;
 	virtual void SendGameMode                   (eGameMode a_GameMode) = 0;
 	virtual void SendHealth                     (void) = 0;
+	virtual void SendHideTitle					(void) = 0;
```

Comment:
Only the at the start of a line you align the code using tabs. After that you use spaces.